test/karma-base.config.babel.js

```diff
@@ -59,7 +59,9 @@ export default {
 
   singleRun: true,
 
-  webpack: webpackConfig,
+  webpack: Object.assign(webpackConfig, {
+    devtool: 'inline-source-map'
```

Comment:
I'm coming back around on sourcemaps. I'll add a task to debug w/o them in the future, but it's pretty annoying to debug webpack w/o them due to the noisy output.